superset-frontend/src/explore/components/controls/AnnotationLayerControl/index.tsx

```diff
@@ -232,27 +275,38 @@ class AnnotationLayerControl extends React.PureComponent {
   }
 }
 
-AnnotationLayerControl.propTypes = propTypes;
-AnnotationLayerControl.defaultProps = defaultProps;
-
 // Tried to hook this up through stores/control.jsx instead of using redux
 // directly, could not figure out how to get access to the color_scheme
-function mapStateToProps({ charts, explore }) {
+function mapStateToProps({
+  charts,
+  explore,
+}: {
+  charts: {
+    [key: string]: ChartState;
+  };
+  explore: Partial<ExplorePageState['explore']>;
+}) {
```

Comment:
You can also use `ExplorePageState['charts']` for the charts type hence it can be like

```suggestion
}: Pick<ExplorePageState, 'charts' | 'explore'>) {
```